### ISSUE TITLE
dependency problem

### ISSUE THREAD

**Issue by JohnHerry** [brentspell/torch-yin#2]
The torch-yin requires numpy version over 1.20,  But I am using python3.6 and it support upto 1.19.5. so my pip install on this package will always fail. Is there any solutions? can you please loose the requirements for dependency?

**Comment by brentspell** [comment]
Hi John. I don't think there was a specific reason for requiring Numpy 1.20, so I lowered it to 1.19 and released version 0.1.3 with this change. The tests pass with NumPy 1.19, but let me know if you run into any problems. Thanks for the feedback.

**Comment by JohnHerry** [comment]
> Hi John. I don't think there was a specific reason for requiring Numpy 1.20, so I lowered it to 1.19 and released version 0.1.3 with this change. The tests pass with NumPy 1.19, but let me know if you run into any problems. Thanks for the feedback.

Thank you for your help,  I was using Python3.6.5 and it support numpy upto 1.19.5, so the directly install of  version 0.1.2  with pip is inpossible, I had tried to get your source code here, and changed the setup.py to lower the requirement of numpy version. But another problem happend on the torch. My local torch version is  1.7.1, It seems that your torch version is higher then 1.8, your code in yin.py: 94

```
   fft = torch.fft.rfft(frames, fft_size, dim=-1)
   corr = torch.fft.ifft(fft * fft.conj())[..., :tau_max]
```

It would not work when torch version is lower then 1.8 . so in the setup.py, it requires torch version only higher then 1.0, that is not enough.

I had tried to change the code from `torch.fft.rfft` to `torch.rfft` in torch1.7, but it still did not work, because the old version of torch.rfft does not support the specified argument `dim=-1`.

At last I have used my pyenv to install a virtualenv of python3.8, with higher numpy and torch version,  and then installed the torchyin with pip.
